### ISSUE TITLE
Apktool v2.3.4 -  has no default translation

### ISSUE THREAD

**Issue by 0pLuS0** [iBotPeaches/Apktool#1958]
### Information
1. **Apktool Version (`apktool -version`)** - 2.3.4
2. **Operating System (Mac, Linux, Windows)** - Linux
3. **APK From? (Playstore, ROM, Other)** - ROM - OxygenOS 5.1.7

### Steps to Reproduce
1. apktool d framework-res.apk
2. apktool b -c framework-res

I've been testing booting up a decompiled/complied framework-res.apk, because I realized I needed to sign it before it could boot, so in this process, when I compile it back, I run into these messages;

Could not find sources is my understanding is normal, as framework-res.apk is the source of all apk, is this a normal message? Then since I'm only running the 2 commands I get all these ```has no default translation``` is this ok, or a problem?

>./apktool b -c framework-res 
I: Using Apktool 2.3.4
W: Could not find sources
I: Checking whether resources has changed...
I: Building resources...
W: warning: string 'BIND_OEMEX_SERVICE' has no default translation.
W: warning: string 'adb_active_generic_notification_message' has no default translation.
W: warning: string 'adb_both_active_notification_title' has no default translation.
W: warning: string 'adb_net_active_notification_title' has no default translation.
W: warning: string 'close' has no default translation.
W: warning: string 'color_picker_blue_label' has no default translation.
W: warning: string 'color_picker_champagne_label' has no default translation.
W: warning: string 'color_picker_default' has no default translation.
W: warning: string 'color_picker_flamingo_label' has no default translation.
W: warning: string 'color_picker_gray_label' has no default translation.
W: warning: string 'color_picker_green_label' has no default translation.
W: warning: string 'color_picker_message_default' has no default translation.
W: warning: string 'color_picker_orange_label' has no default translation.
W: warning: string 'color_picker_pear_label' has no default translation.
W: warning: string 'color_picker_pink_label' has no default translation.
W: warning: string 'color_picker_purple_label' has no default translation.
W: warning: string 'color_picker_red_label' has no default translation.
W: warning: string 'color_picker_turquoise_label' has no default translation.
W: warning: string 'color_picker_unknown_label' has no default translation.
W: warning: string 'color_picker_yellow_label' has no default translation.
W: warning: string 'ext_media_ready_notification_title' has no default translation.
W: warning: string 'global_action_reboot' has no default translation.
W: warning: string 'global_action_screenshot' has no default translation.
W: warning: string 'high_definition_tag' has no default translation.
W: warning: string 'install_fail_oplib_not_satisfied' has no default translation.
W: warning: string 'kg_fingerprint_verification_failed' has no default translation.
W: warning: string 'lock_to_app_description_hardware' has no default translation.
W: warning: string 'lock_to_app_toast_hardware' has no default translation.
W: warning: string 'op_applying' has no default translation.
W: warning: string 'oxygen_system_label' has no default translation.
W: warning: string 'permdesc_modifyWavesSettings' has no default translation.
W: warning: string 'permdesc_setKeyguardWallpaper' has no default translation.
W: warning: string 'permdesc_switchMaterialTheme' has no default translation.
W: warning: string 'permlab_modifyWavesSettings' has no default translation.
W: warning: string 'permlab_setKeyguardWallpaper' has no default translation.
W: warning: string 'permlab_switchMaterialTheme' has no default translation.
W: warning: string 'power_intensive_notificaton_string' has no default translation.
W: warning: string 'reboot' has no default translation.
W: warning: string 'reboot_bootloader' has no default translation.
W: warning: string 'reboot_confirm' has no default translation.
W: warning: string 'reboot_progress' has no default translation.
W: warning: string 'reboot_recovery' has no default translation.
W: warning: string 'reboot_title' has no default translation.
W: warning: string 'reset_retail_demo_mode_text' has no default translation.
W: warning: string 'reset_retail_demo_mode_title' has no default translation.
W: warning: string 'top_app_killed_message' has no default translation.
W: warning: string 'top_app_killed_title' has no default translation.
W: warning: string 'wfcSpnFormat' has no default translation.
W: warning: string 'zzz_op_global_action_confirm_poweroff' has no default translation.
W: warning: string 'zzz_op_global_action_confirm_reboot' has no default translation.
I: Copy original files...
I: Copy AndroidManifest.xml...
I: Copy META-INF...
I: Building apk file...
I: Copying unknown files/dir...
I: Built apk...

### Frameworks
If this APK is from an OEM ROM (Samsung, HTC, LG). Please attach framework files
(`.apks` that live in `/system/framework` or `/system/priv-app`)

[framework-res.apk.zip](https://github.com/iBotPeaches/Apktool/files/2674102/framework-res.apk.zip)

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything?
Yes, as I mentioned above, I only tested this by only running two commands and then signing it to make sure it will boot;
1. apktool d framework-res.apk
2. apktool b -c framework-res

2. If you are trying to install a modified apk, did you resign it?
Yes
java -jar signapk.jar certificate.pem key.pk8 framework-res.apk test.apk

3. Are you using the latest apktool version? Yes - 2.3.4

**Comment by LynxKik** [comment]
Pretty sure this is because a String exists in a `values-language` folder but not the actual `values` folder

**Comment by 0pLuS0** [comment]
Well all I did was these cmds;

apktool d framework-res.apk

Then edited the framework for what I wanted to do, which was simple change in /valuesbool.xml, I changed two lines that were listed as false to true, then compiled it back

apktool b -c framework-res

Then I signed it and put it back into the ROM and the support I wanted out of it is working, the ROM seems to run fine, but if this is not correct I'd like to fix it, but I have not idea here.

If I pasted the framework-res.apk, would you know what is going on, how to fix this?

Thanks

**Comment by 0pLuS0** [comment]
Well all I did was these cmds;

apktool d framework-res.apk

Then edited the framework for what I wanted to do, which was in /values/bool.xml, I changed two lines that were listed as false to true, then compiled it back

apktool b -c framework-res

Then I signed it and put it back into the ROM and the support I wanted out of it is working, the ROM seems to run fine.

If I just run these commands only, and make no changes; 

apktool d framework-res.apk
apktool b framework-res
or 
apktool b -c framework-res

I still see this output; ```has no default translation.```

I'm assuming just running these commands is correct and doesn't cause anything like this, and it's just simply the way this framework-res.apk came from OnePlus, it's just something on their part they did, or forgot to do etc...?

Thanks

**Comment by 0pLuS0** [comment]
@iBotPeaches ok, but as I mentioned, all did was decompile it, then compile it back with no changes, so I assumed that meant this is just the way it is from OpenPlus who created it?

Thanks

**Comment by iBotPeaches** [comment]
That is correct. Many applications are built with outdated tools or simply ignore warnings.

**Comment by 0pLuS0** [comment]
Those BASTARDS! LOL

Makes ya think they don't know what they are doing...

Happy Holidays
